packages/react-atlas-core/webpack.config.js

```diff
@@ -11,6 +11,21 @@ module.exports = {
     publicPath: '/lib/',
     libraryTarget: 'commonjs2'
   },
+  externals: [{
```

Comment:
But maybe after V4.0.0 of bs table library will remove react-toastr dependency (which is the reason to depends on old react version in bs table)

https://github.com/AllenFang/react-bootstrap-table/pull/1216
https://github.com/AllenFang/react-bootstrap-table/issues/863